docs/csharp/quick-starts/numbers-in-csharp-local.md

```diff
@@ -0,0 +1,373 @@
+
+```csharp
+double a = 5;
+double b = 4;
+double c = 2;
+double d = (a  + b) / c;
+Console.WriteLine(d);
+```
+
+Notice that the answer includes the decimal portion of the quotient. Try a slightly
+more complicated expression with doubles:
+
+```csharp
+double e = 19;
+double f = 23;
+double g = 8;
+double h = (e  + f) / g;
+Console.WriteLine(h);
+```
+
+The range of a double value is much greater than integer values. Try the following
+code below what you've written so far:
+
+```csharp
+double max = double.MaxValue;
+double min = double.MinValue;
+Console.WriteLine($"The range of double is {min} to {max}");
+```
+
+These values are printed out to represent scientific notation. The number to
+the left of the `E` is the significant. The number to the right is the exponent,
+as a power of 10. Internally, these numbers are represented similarly, but in 
+a binary form. One part represents the significant binary digits.
+Another part represents where to place the binary point (analogous to a decimal point) relative to
+those significant digits.
+
+Just like decimal numbers in math, doubles in C# can have rounding errors. Try this code:
+
+```csharp
+double third = 1.0 / 3.0;
+Console.WriteLine(third);
+```
+
+You know that `0.3` repeating is not exactly the same as `1/3`.
+
+***Challenge***
+
+Try other calculations with large numbers, small numbers, multiplication
+and division using the `double` type.  Try more complicated calculations.
+
+After you've spent some time with the challenge, take the code you've written
+and place it in a new method. Name that new method `WorkWithDoubles`.
+
+## Work with fixed point types
+
+You've seen the basic numeric types in C#: integers and doubles.  There is one
+other type to learn: the `decimal` type. The `decimal` type has a smaller
+range, but greater precision than `double`. The term **fixed point** means
+that the decimal point (or binary point) doesn't move. Let's take a look:
+
+```csharp
+decimal min = decimal.MinValue;
+decimal max = decimal.MaxValue;
+Console.WriteLine($"The range of the decimal type is {min} to {max}");
+```
+
+Notice that the range is smaller than the `double` type. You can see the greater
+precision with the decimal type by trying the following code:
+
+```csharp
+double a = 1.0;
+double b = 3.0;
+Console.WriteLine(a / b);
+
+decimal c = 1.0M;
+decimal d = 3.0M;
+Console.WriteLine(c / d);
+```
+
+The `M` suffix on the numbers is how you indicate that a constant should use the
+`decimal` type.
+
+Notice that the math using the decimal type has more digits to the right
+of the decimal point. 
+
+***Challenge***
+
+Now that you've seen the different numeric types, write code that calculates
+the area of a circle whose radius is 2.50 inches. Remember that the area of a circle
+is the radius squared multiplied by PI. One hint: C# contains a constant
+for PI: `Math.PI` that you can use for that value. 
```

Comment:
PI: `Math.PI` that --> PI, `Math.PI`, that
possibly use a link (\<xref:System.Math.PI?displayProperty=nameWithType>)?